src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -727,7 +709,7 @@ export class MeetingManager implements AudioVideoObserver {
     this.selectedVideoInputTransformDeviceObservers.push(callback);
   };
 
-  unsubscribeFromSelectedVideoInputTranformDevice = (
+  unsubscribeFromSelectedVideoInputTransformDevice = (
```

Comment:
So we still have the notion of separate handling of transform device and normal device right?

Reply:
For current PR, yes. The `deviceId` type change is under development.